commands/unequip.js

```diff
@@ -0,0 +1,129 @@
+const funcall = require("../modules/funcall.js");
+//simple ping command to check if the bot is online.
+const strifecall = require("../modules/strifecall.js");
+exports.type = "sylladex";
+exports.desc = "Unequip a weapon or other equipment"
+exports.use = `";unequip" removes your currently-equipped weapon from your specibus and puts it in your sylladex.
+";unequip [number]" unequips the indicated weapon from your specibus.
+";unequip trinket" unequips your equipped trinket instead.
+";unequip armor" unequips your equipped armor instead.`
+exports.run = (client, message, args) => {
+	var userid = message.guild.id.concat(message.author.id);
+	var charid = client.userMap.get(userid,"possess");
+
+	let sdex = client.charcall.charData(client,charid,"sdex");
+	let scards = client.charcall.charData(client,charid,"scards");
+	let spec = client.charcall.charData(client,charid,"spec");
+	let kinds = client.charcall.charData(client,charid,"kinds");
+
+	var trinket = client.charcall.charData(client,charid,"trinket");
+	var armor = client.charcall.charData(client,charid,"armor");
+	
+	let curWeapon = client.charcall.charData(client,charid,"equip");
+
+	if(!args[0])
+	{
+		//make sure strife specibus is allocated
+		if(kinds.length == 0){
+			message.channel.send(`Your STRIFE SPECIBUS is not currently allocated to a weaponkind, you must ${client.auth.prefix}allocate it before you can ${client.auth.prefix}equip a weapon!`);
+			return;
+		};
+		
+		args[0] = "" + curWeapon;
+	}
+
+	let mess = `Successfully UNEQUIPPED the `;
+	let unequipItem;
+
+	switch(args[0].toUpperCase())
+	{
+		case "TRINKET":
+		{
+			if(trinket == "NONE")
+			{
+				message.channel.send(`You don't have a trinket equipped!`);
+				return;
+			}
+			unequipItem = trinket.splice(0,1)[0];
+			
+			mess += `${unequipItem[0]} from your TRINKET SLOT`;
+		}
+		break;
+		
+		case "ARMOR":
+		case "ARMOUR":
+		{
+			if(armor == "NONE")
+			{
+				message.channel.send(`You don't have armor equipped!`);
+				return;
+			}
+			unequipItem = armor.splice(0,1)[0];
+
+			mess += `${unequipItem[0]} from your ARMOR SLOT`;
+		}
+		break;
+		
+		default:
+		{
+			if(kinds.length == 0){
```

Comment:
You already make this check above, any reason you check it twice?

Reply:
That one was for when args[0] doesn't exist, while this one is when args[0] _does_ exist, but the player still plans to unequip something from the Strife Specibus.

Reply:
well you could still allow people to eject strife cards before they allocate a weaponkind, but is there a check for if they have no strife cards to eject?